cmd/nerdctl/image_convert.go

```diff
@@ -67,6 +68,16 @@ func newImageConvertCommand() *cobra.Command {
 	imageConvertCommand.Flags().Bool("zstdchunked", false, "Use zstd compression instead of gzip (a.k.a zstd:chunked). Should be used in conjunction with '--oci'")
 	// #endregion
 
+	// #region nydus flags
+	imageConvertCommand.Flags().Bool("nydus", false, "Convert an OCI image to Nydus image. Should be used in conjunction with '--oci'")
+	imageConvertCommand.Flags().String("nydus-builder-path", "nydus-image", "The nydus-image binary path, if unset, search in PATH environment")
+	imageConvertCommand.Flags().String("nydus-work-dir", "/tmp", "Work directory path for image conversion")
+	imageConvertCommand.Flags().String("nydus-chunk-dict", "", "Specify the path of the chunk dictionary")
+	imageConvertCommand.Flags().String("nydus-prefetch-patterns", "", "The file path pattern list want to prefetch")
+	imageConvertCommand.Flags().String("nydus-compressor", "lz4_block", "The algorithm to compress image data blob, default: lz4_block, possible values: [none, lz4_block, gzip, zstd]")
+	imageConvertCommand.Flags().String("nydus-fs-version", "5", "Version number of nydus image format, possible values: 5, 6")
```

Comment:
Any reason to prefer 5 over 6?

Reply:
Using version 5 in the current production environment is fine.
Version 6 is an EROFS compatible format which has performance advantages, and it requires linux kernel version >= 5.19 to fully utilize the performance advantages.